plugins/flatpak/gs-self-test.c

```diff
@@ -1483,6 +1483,113 @@ gs_plugins_flatpak_app_update_func (GsPluginLoader *plugin_loader)
 	g_assert_cmpint (gs_app_get_state (app_source), ==, AS_APP_STATE_AVAILABLE);
 }
 
+static void
+gs_plugins_flatpak_runtime_extension_func (GsPluginLoader *plugin_loader)
+{
+	GsApp *app;
+	GsApp *extension;
+	gboolean ret;
+	g_autofree gchar *repodir1_fn = NULL;
+	g_autoptr(GError) error = NULL;
+	g_autoptr(GsApp) app_source = NULL;
+	g_autoptr(GsAppList) list = NULL;
+	g_autoptr(GsAppList) list_updates = NULL;
+	g_autoptr(GsPluginJob) plugin_job = NULL;
+	g_autoptr(GMainLoop) loop = g_main_loop_new (NULL, FALSE);
+	GsAppList *global_cache = NULL;
+
+	/* drop all caches */
+	gs_plugin_loader_setup_again (plugin_loader);
+
+	/* no flatpak, abort */
+	if (!gs_plugin_loader_get_enabled (plugin_loader, "flatpak"))
```

Comment:
You could call `g_assert_true()` here instead

Reply:
This was copied from previous code by Richard, but of course I can change it.